cmd/podman/images.go

```diff
@@ -302,6 +303,14 @@ func getImagesTemplateOutput(ctx context.Context, runtime *libpod.Runtime, image
 					Size:        sizeStr,
 				}
 				imagesOutput = append(imagesOutput, params)
+				if opts.quiet {
+					breakOuter = true
```

Comment:
could you tag the loop then use `break tag` instead?

```
outer:	for repo, tags := range image.ReposToMap(img.Names()) {
                     for _, tag := range tags {
.....
                          if opts.quiet {
                               break outer;
                           }
....
        }
}
```

then you won't need the other `if` block

Reply:
I'd not considered that, a very good thought @giuseppe , I'll give it a whirl.